### PR TITLE
complete controller description

### PR DESCRIPTION
**Pull request by lance5890**
complete controller description for cluster-policy-ontroller

### CODE REVIEW

**Comment by lance5890** [comment]
@ingvagabund

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ The cluster-policy-controller is responsible for maintaining policy resources ne
 Controllers managed by cluster-policy-controller are:
 * cluster quota reconcilion - manages cluster quota usage
 * namespace SCC allocation controller - allocates UIDs and SELinux labels for namespaces     
+* cluster csr approver controller - csr approver for monitoring scraping
+* podsecurity admission label syncer controller - configure the PodSecurity admission namespace label
```
Comment (atiratree) [inline README.md:7]: maybe we could be a little bit more specific?

```suggestion
* podsecurity admission label syncer controller - configure the PodSecurity admission namespace label for namespaces with "security.openshift.io/scc.podSecurityLabelSync: true" label
```

**Pull request merged**